src/main/java/net/fabricmc/loader/gui/FabricStatusTree.java

```diff
@@ -204,9 +204,32 @@ public void setInfo() {
 		}
 
 		public FabricStatusNode addChild(String string) {
-			FabricStatusNode child = new FabricStatusNode(this, string);
-			children.add(child);
-			return child;
+			if (string.startsWith("\t")) {
+				if (children.size() == 0)
+					throw new RuntimeException("Tried to add indented node without having a root node");
+				FabricStatusNode lastChild = children.get(children.size() - 1);
+				FabricStatusNode subChild = new FabricStatusNode(lastChild, emboldenForNode(string.substring(1)));
+				subChild.warningLevel = FabricTreeWarningLevel.INFO;
+				lastChild.children.add(subChild);
+				lastChild.expandByDefault = true;
+				return lastChild;
+			} else {
+				FabricStatusNode child = new FabricStatusNode(this, cleanForNode(string));
+				children.add(child);
+				return child;
+			}
+		}
+
+		private String cleanForNode(String string) {
+			int dashIndex = string.indexOf('-');
+			if (dashIndex > 0)
+				string = string.substring(dashIndex + 1);
+			string = string.trim();
+			return string;
```

Comment:
This should probably only remove the dash if the string *starts* with a dash rather than just finding the first one to remove.

Reply:
whoops